openassessment/xblock/static/js/spec/lms/oa_staff_area.js

```diff
@@ -200,8 +202,8 @@ describe('OpenAssessment.StaffAreaView', function() {
         it('tracks unsubmitted assessments in multiple views', function() {
             var fullGradeStaffArea = createStaffArea({}, 'oa_staff_area_full_grading.html'),
                 staffOverrideStaffArea = createStaffArea(),
-                fullGradeTab = "staff-grading",
-                staffOverrideTab = "staff-tools",
+                fullGradeTab = 'staff-grading',
```

Comment:
Same question: " vs ' - what's the difference, if any?

Reply:
While either `"` or `'` works, `eslint` is configured to warn about `"`.